src/tools/pub/cmd/pub-outdated.md

```diff
@@ -0,0 +1,251 @@
+---
+title: pub outdated
+description: Use pub outdated to help you update your package dependencies.
+diff2html: true
+---
+
+_Outdated_ is one of the commands of the [pub tool](/tools/pub/cmd).
+
+{% prettify nocode tag=pre+code %}
+$ pub outdated [options]
+{% endprettify %}
+
+Use `pub outdated` to identify out-of-date [package dependencies][]
+and get advice on how to update them.
+<span style="color:red">[PENDING: Should we motivate why you want to do this?
```

Comment:
Yes, using latest stable versions is good guidance to ensure you have the latest bug fixes, and other improvements.